src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class Calculator extends React.PureComponent {
```

Comment:
- [ ] Instead of this being a class component, you are required to make it a function-based component.